pkg/wolfssl/sock_tls/sock_tls.c

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright (C) 2019 Daniele Lacamera
+ *
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+#include <string.h>
+#include <stdlib.h>
+#include <stdio.h>
+
+#include <net/sock.h>
+#include <wolfssl/ssl.h>
+#include <wolfssl/internal.h>
+
+void sock_dtls_close(sock_tls_t *sk)
+{
+    sock_udp_close(&sk->conn.udp);
+}
+
+void sock_dtls_set_endpoint(sock_tls_t *sk, const sock_udp_ep_t *addr)
+{
+    printf("wolfSSL: Setting peer address and port\n");
+    memcpy(&sk->peer_addr, addr, sizeof (sock_udp_ep_t));
```

Comment:
I don't get why here `memcpy()` is used and later `XMEMCPY()` is used. My gut feeling is that this is unintentional. (Remark, not insisting on changes.)

Reply:
Well spotted